src/pages/Login.vue

```diff
@@ -281,14 +281,14 @@ export default {
             this.$recaptcha("login")
               .then((recaptchaToken) => {
                 this._register(idToken, name, this.referral, recaptchaToken)
```

Comment:
Why not this?
```suggestion
                return this._register(idToken, name, this.referral, recaptchaToken)
```
I think promise chaining would suit this use-case